generators/templates.go

```diff
@@ -87,14 +87,70 @@ package client
 
 import (
 	"bytes"
+	"crypto/tls"
 	"encoding/json"
 	"errors"
 	"io/ioutil"
+	"os"
+	"os/user"
 	"net/http"
 
 	log "github.com/Sirupsen/logrus"
 )
 
+var httpClient *http.Client
+
+func init() {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
+	httpClient = &http.Client{Transport: tr}
+}
+
+type NetctlConfig struct {
+	Token string ` + "`" + `json:"token"` + "`" + `
+}
```

Comment:
Makes sense to me!

This was a quick hack to see if I could add `netctl login` functionality in a reasonable amount of time without breaking anything.  It works, now it just needs to be done in an acceptable manner.  Any architecture suggestions you have would be valuable